docs/book/component-guide/step-operators/vertex.md

```diff
@@ -115,13 +115,13 @@ For additional configuration of the Vertex step operator, you can pass `VertexSt
 from zenml import step
 from zenml.integrations.gcp.flavors.vertex_step_operator_flavor import VertexStepOperatorSettings
 
-@step(step_operator= <NAME>, settings={"step_operator.vertex": VertexStepOperatorSettings(
```

Comment:
Not sure we should fix that to a name here, people might think they can just copy-paste. the settings key is always fixed which is why we can hard code it here.